packages/cli/generators/app/templates/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:10
+FROM node:10.15-slim
```

Comment:
Let's use `node:10-slim` to pick up latest 10.x version.

Reply:
Any reason change the variant to `slim`?

Reply:
@jannyHou , the original setting (node:10) had security and configuration issues detected by IBM Cloud's vulnerability advisor (after pushing image to IBM Cloud registry). It was also 424 MB). The new 'slim' value is 131 MB.

Reply:
To avoid confusions, we can probably add some comments to the generated `Dockerfile`.

Reply:
@raymondfeng , ok I will try again as node:10-slim ...to see if there are any security or config issues....

Reply:
@raymondfeng and I decided to use the dockerhub url instead: https://hub.docker.com/_/node